VTEX - Orders API.json

```diff
@@ -20503,7 +20143,7 @@
                                 "description": "Quantity of items been replaced."
```

Comment:
```suggestion
                                "description": "Quantity of items replaced."
```

Reply:
This is about an order change that was already made, maybe even a long time ago, so I will keep "replaced" instead of "been replaced".